test.md

```diff
@@ -0,0 +1 @@
+This file should be ignored by git.
```

Comment:
- [ ] The `test.md` file shouldn't be tracked by Github. Kindly untrack the file.